sentry-ruby/lib/sentry/transport/http_transport.rb

```diff
@@ -148,6 +151,9 @@ def conn
       connection
     end
 
+    # @param proxy [String, URI, Hash] Proxy config value passed into `config.transport`.
+    #   Accepts either a URI formatted string, URI, or a hash with the `uri`, `user`, and `password` keys.
+    # @return [Hash] Normalized proxy config that will be passed into `Net::HTTP`
```

Comment:
I don't think it's necessary to document this method with YARD as it's internal to the SDK.

Maybe it's better to document [`Transport::Configuration#proxy`](https://github.com/getsentry/sentry-ruby/blob/master/sentry-ruby/lib/sentry/transport/configuration.rb#L6)instead?

Reply:
Yep, seemed like transport configuration could benefit from some documentation: https://github.com/getsentry/sentry-ruby/pull/2161/commits/d97e181c182aeb4750481a3ce343d9002f488e17